test/CadlRanchProjects/client/structure/renamed-operation/tests/Generated/Samples/Samples_Group.cs

```diff
@@ -32,8 +32,8 @@ public void Example_Group_RenamedTwo_ShortVersion()
         [Ignore("Only validating compilation of examples")]
         public async Task Example_Group_RenamedTwo_ShortVersion_Async()
         {
-            Uri endpoint = new Uri("<https://my-service.azure.com>");
-            Group client = new RenamedOperationClient(endpoint, ClientType.Default).GetGroupClient();
+            Uri endpoint = new Uri("<endpoint>");
+            Group client = new RenamedOperationClient(endpoint, default).GetGroupClient();
```

Comment:
This change because the clientType is client parameter, and it is propagate to operation, so there is no mocked value for it.